sdk/face/ai-vision-face-rest/src/models.ts

```diff
@@ -1,67 +1,134 @@
+  sendResultsToClient?: boolean;
+  /** Whether or not to allow client to set their own 'deviceCorrelationId' via the Vision SDK. Default is false, and 'deviceCorrelationId' must be set in this request body. */
+  deviceCorrelationIdSetInClient?: boolean;
+  /** Whether or not store the session image. */
+  enableSessionImage?: boolean;
+  /**
+   * The model version used for liveness classification. This is an optional parameter, and if this is not specified, then the latest supported model version will be chosen
+   *
+   * Possible values: "2022-10-15-preview.04", "2023-12-20-preview.06"
+   */
+  livenessSingleModalModel?: LivenessModel;
+  /** Unique Guid per each end-user device. This is to provide rate limiting and anti-hammering. If 'deviceCorrelationIdSetInClient' is true in this request, this 'deviceCorrelationId' must be null. */
+  deviceCorrelationId?: string;
+  /** Seconds the session should last for. Range is 60 to 86400 seconds. Default value is 600. */
+  authTokenTimeToLiveInSeconds?: number;
+  /** Whether or not return the verify image hash. */
+  returnVerifyImageHash?: boolean;
+  /** Threshold for confidence of the face verification. */
+  verifyConfidenceThreshold?: number;
+}
+
 /** Alias for DetectionModel */
-export type DetectionModel = string | "detection_01" | "detection_02" | "detection_03";
+export type DetectionModel = string;
 /** Alias for RecognitionModel */
-export type RecognitionModel =
-  | string
-  | "recognition_01"
-  | "recognition_02"
-  | "recognition_03"
-  | "recognition_04";
+export type RecognitionModel = string;
 /** Alias for FaceAttributeType */
-export type FaceAttributeType =
-  | string
-  | "headPose"
-  | "glasses"
-  | "occlusion"
-  | "accessories"
-  | "blur"
-  | "exposure"
-  | "noise"
-  | "mask"
-  | "qualityForRecognition"
-  | "age"
-  | "smile"
-  | "facialHair"
-  | "hair";
+export type FaceAttributeType = string;
 /** Alias for FindSimilarMatchMode */
-export type FindSimilarMatchMode = string | "matchPerson" | "matchFace";
+export type FindSimilarMatchMode = string;
 /** Alias for LivenessOperationMode */
-export type LivenessOperationMode = string | "Passive" | "PassiveActive";
+export type LivenessOperationMode = string;
+/** Alias for LivenessModel */
+export type LivenessModel = string;
 /** Request of liveness with verify session creation. */
-export type CreateLivenessWithVerifySessionContent =
+export type CreateLivenessWithVerifySessionMultipartContent =
   | FormData
   | Array<
-      | CreateLivenessWithVerifySessionContentParametersPartDescriptor
-      | CreateLivenessWithVerifySessionContentVerifyImagePartDescriptor
+      | CreateLivenessWithVerifySessionMultipartContentParametersPartDescriptor
+      | CreateLivenessWithVerifySessionMultipartContentVerifyImagePartDescriptor
     >;
 /** API versions for Azure AI Face API. */
-export type Versions = "v1.1-preview.1";
+export type Versions = "v1.1-preview.1" | "v1.2-preview.1";
```

Comment:
wonder if we should remove this for now?

Reply:
Do you mean remove `v1.1-preview.1`? It's generated but I think it's OK.